tools/gfx/d3d12/d3d12-shader-object-layout.h

```diff
@@ -66,6 +66,9 @@ class ShaderObjectLayoutImpl : public ShaderObjectLayoutBase
         /// as a sub-object.
         uint32_t subObjectIndex;
 
+        /// The stride of a structured buffer.
+        uint32_t bufferElementStride;
```

Comment:
This variable may be used with an uninitialized value.

Reply:
We actually default initialize the whole struct whenever it is initialized, so this is fine.